[inductor] Test for shape padding

bertmaher opened the pull request:
Summary:
This wasn't tested anywhere as far as I can tell, so it was breaking
on Triton updates that fiddled with the signature of `do_bench`

Test Plan: `test_shape_padding`

Differential Revision: D45499978



cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

Chillee commented:
Fine with me. Btw, I think @rkindi is working on some improvements to this pass.

bertmaher commented:
@pytorchbot merge

bertmaher commented:
@pytorchbot merge

bertmaher committed:
[inductor] Test for shape padding (#100493)

Summary:
This wasn't tested anywhere as far as I can tell, so it was breaking
on Triton updates that fiddled with the signature of `do_bench`

Pull Request resolved: https://github.com/pytorch/pytorch/pull/100493

Test Plan:
`test_shape_padding`

cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

Reviewed By: Chillee

Differential Revision: D45499978

Pulled By: bertmaher

fbshipit-source-id: 386ee6cac4e746b5da5a29b3a0faa3467d656f6f
